packages/hardhat-zksync-solc/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@matterlabs/hardhat-zksync-solc",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
Side comment: I think we should follow the semver. If we only fixed some bugs (and it doesn't really make sense to keep using the old version of the plugin), we should bump the patch version.

We will bump minor, if:
- new API was added.
- some changes were breaking (either code-wise or usage-wise).

Reply:
This was left there from the previous update, but I got you :+1:  
This one will be 0.2.1